cfpb/cfgov-refresh
# College costs: template updates

Updates to college costs templates based on cleared content. Some templates (like the summary and save pages) are still in progress.


## Changes

-  Updated college costs templates
- Updated college costs LESS

## How to test this PR

1. Check out branch
2. Run `yarn gulp styles`
3. Create a page using the `college costs` page type and verify that the hard-coded contents display as expected


## Checklist
